resources/terraform/cloudflare-lbs/main.tf

```diff
@@ -0,0 +1,287 @@
+
+  origin_steering {
+    policy = "random"
+  }
+
+  origins {
+    name    = "rpc-0-taurus"
+    address = "52.91.1.29"
+    enabled = true
+    weight  = 0.5
+  }
+
+  origins {
+    name    = "rpc-1-taurus"
+    address = "174.129.155.116"
+    enabled = true
+    weight  = 0.5
+  }
+
+}
+
+# Pool 3: Taurus-RPC-EVM
+resource "cloudflare_load_balancer_pool" "taurus_rpc_evm" {
+  account_id      = var.cloudflare_account_id
+  name            = "Taurus-RPC-EVM"
+  description     = ""
+  enabled         = true
+  minimum_origins = 1
+  monitor         = cloudflare_load_balancer_monitor.rpc_taurus_evm_health_check.id
+  check_regions   = ["WNAM"]
+
+  origin_steering {
+    policy = "random"
+  }
+
+  origins {
+    name    = "auto-evm-1"
+    address = "18.234.222.92"
+    enabled = true
+    weight  = 0.5
+  }
+
+  origins {
+    name    = "auto-evm-3"
+    address = "65.108.232.15"
+    enabled = false
+    weight  = 0.5
+  }
+
+}
+
+# Pool 4: Taurus-RPC-EVM-Fallback
+resource "cloudflare_load_balancer_pool" "taurus_rpc_evm_fallback" {
+  account_id      = var.cloudflare_account_id
+  name            = "Taurus-RPC-EVM-Fallback"
+  description     = "Taurus Auto-EVM RPC LB"
+  enabled         = true
+  minimum_origins = 1
+  monitor         = cloudflare_load_balancer_monitor.rpc_taurus_evm_health_check.id
+  check_regions   = ["WNAM"]
+
+  origins {
+    name    = "auto-evm-0"
+    address = "34.238.40.85"
+    enabled = true
+    weight  = 0.25
+  }
+
+  origins {
+    name    = "auto-evm-2"
+    address = "65.108.232.16"
+    enabled = false
+    weight  = 0.75
+  }
+
+}
+
+# =============================================================================
+# LOAD BALANCERS (Import these last)
+# =============================================================================
+
+# Load Balancer 1: auto-evm-lb.taurus.autonomys.xyz
+resource "cloudflare_load_balancer" "auto_evm_lb_taurus" {
+  zone_id          = var.cloudflare_zone_id
+  name             = "auto-evm-lb.taurus.autonomys.xyz"
+  description      = ""
+  proxied          = true
+  enabled          = true
+  session_affinity = "none"
+  steering_policy  = "random"
+  fallback_pool_id = cloudflare_load_balancer_pool.taurus_rpc_evm_fallback.id
+
+  default_pool_ids = [
+    cloudflare_load_balancer_pool.taurus_rpc_evm.id,
+    cloudflare_load_balancer_pool.taurus_rpc_evm_fallback.id
+  ]
+
+  session_affinity_attributes {
+    samesite               = "Auto"
+    secure                 = "Auto"
+    drain_duration         = 180
+    zero_downtime_failover = "sticky"
+  }
+
+  adaptive_routing {
+    failover_across_pools = true
+  }
+
+  random_steering {
+    default_weight = 1
+  }
+
+  location_strategy {
+    prefer_ecs = "proximity"
+    mode       = "pop"
+  }
+
+}
+
+# Load Balancer 2: rpc-lb.mainnet.autonomys.xyz
+resource "cloudflare_load_balancer" "rpc_lb_mainnet" {
+  zone_id          = var.cloudflare_zone_id
+  name             = "rpc-lb.mainnet.autonomys.xyz"
```

Comment:
does this mean we can use `rpc-lb.mainnet.autonomys.xyz` for polkadot js or as rpc url ?

Does individual node instaces running Traefik need to generate cert for this or Cloudflare changes origin to `rpc-1.mainnet.subspace.network` ?